addons/sale/views/sale_views.xml

```diff
@@ -332,7 +332,7 @@
                                 </div>
                                 <field name="amount_total" nolabel="1" class="oe_subtotal_footer_separator" widget='monetary' options="{'currency_field': 'currency_id'}"/>
                             </group>
-                            <field name="note" class="oe_inline" placeholder="Setup default terms and conditions in your company settings."/>
+                            <field name="note" class="oe_inline" placeholder="Setup default terms and conditions in your Sales Settings ..."/>
```

Comment:
remove the capital S for sales and settings. Thank you.
It should be "Setup default terms and conditions in your sales settings ..."